### DISCUSSION TITLE
Publishing npm package to GitHub registry disabled for account

### DISCUSSION THREAD

**Discussion by anthonyec** [community/community#29566] [Actions]
Hello! I’m trying to publish a NodeJS package following the GitHub instructions found on their [homepage](https://github.com/features/packages) and [docs](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry). But when I run `npm publish` I get the following error:

```txt
npm ERR! code E403
npm ERR! 403 403 Forbidden - PUT https://npm.pkg.github.com/@anthonyec%2ftest_package - The "anthonyec" account 
is disabled and can't accept package uploads.
```

I'm running this on an [isolated test repo](https://github.com/anthonyec/test_package), using the publish command locally, not via GH actions (though GH actions is how I first encountered the problem). I'm also successfully logged in using `npm login` with GH credentials.

I've seen the following issues:
- [GitHub Actions is currently disabled for your account](https://github.com/orgs/community/discussions/19015)
- [Request GitHub Package Registry beta for kubernetes-sigs org](https://github.com/kubernetes/org/issues/812)

But I don't think those issues are quite the same. The only relevant [Stack Overflow question](https://stackoverflow.com/questions/63550967/error-403-forbidden-account-disabled-when-publishing-npm-package) has no answers sadly. 

Has anyone seen this issue before with publishing? Am I missing something obvious?

**Comment by mazkdevf** [comment]
Are you sure your permissions are set up the correct way?

**Reply by anthonyec** [reply]
I _think_ so. My personal access token used locally to login has the permissions to read and write packages. Along with full repo access.

<img width="784" alt="image" src="https://user-images.githubusercontent.com/1451668/184697716-22990a05-9b4d-4257-b21d-7d50f7464321.png">

Is there additional permissions stuff I might be missing?

**Reply by mazkdevf** [reply]
Hmm, I'm not really sure, do you have 2FA Enabled on NPM Account*?

**Reply by anthonyec** [reply]
Yup, 2FA enabled

**Reply by anthonyec** [reply]
Though this isn't using my NPM account. I'm trying to use the GH registry instead of NPM

**Reply by mazkdevf** [reply]
Okay umm, If its all the time saying it is locked, I think probably best to do is to create an Ticket from: **https://support.github.com/request**

**Reply by anthonyec** [reply]
I'll give that a try also, thanks!

**Reply by mazkdevf** [reply]
> I'll give that a try also, thanks!

np, have a good day!
